src/webapp/components/add-new-option/AddNewOption.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styled from "styled-components";
+import { AddCircleOutline } from "@material-ui/icons";
+
+import i18n from "../../../utils/i18n";
+
+type AddNewOptionProps = {
+    id: string;
+    label?: string;
+    onAddNewOption: () => void;
+};
+
+export const AddNewOption: React.FC<AddNewOptionProps> = React.memo(
+    ({ id, label, onAddNewOption }) => {
```

Comment:
[minor] [subjective] If we de-structure the props in the first line of the block, we remove 1 indentation level.

Reply:
You're right! I will have this in mind for next components